### PR TITLE
Change entities to 2e ED, and update TTML1 bibref.

### PR DESCRIPTION
**Pull request by skynavga**

### CODE REVIEW

File: spec/entitiesedcopy.dtd [hunk 1]
```diff
@@ -15,16 +15,16 @@
...
 
 <!ENTITY w3c.tr  "">	
 
 <!ENTITY document.role "editors-copy">
 
 <!-- The following is used for document .-->
-<!ENTITY document.status "W3C Proposed Recommendation">
-<!ENTITY w3c.status "PR">
-<!ENTITY doctype "pr">
+<!ENTITY document.status "W3C Working Draft">
```
Comment (nigelmegitt) [inline spec/entitiesedcopy.dtd]: Shouldn't this be Editors Draft? I see that for WDs we have `entitieswd.dtd` available at line 4, in place of `entitiesedcopy.dtd`.

File: spec/entitiesedcopy.dtd [hunk 2]
```diff
@@ -15,16 +15,16 @@
...
+<!ENTITY w3c.status "WD">
+<!ENTITY doctype "wd">
```
Comment (nigelmegitt) [inline spec/entitiesedcopy.dtd]: Likewise shouldn't the above two entries be `ED` and `ed`, respectively?

File: spec/entitiesedcopy.dtd (hunk 1 above)
Comment (skynavga) [reply spec/entitiesedcopy.dtd]: No. It is an editor's draft of a WD, so we label it as WD, but the SoTD states "This document is an editor's copy that has no official standing," and the label on left of screen shows "W3C Editor's Draft". In actuality, I think we don't actually need (or use) `entitiesedcopy.dtd`.

File: spec/entitiesedcopy.dtd (hunk 2 above)
Comment (skynavga) [reply spec/entitiesedcopy.dtd]: See above.

File: spec/entitiesedcopy.dtd (hunk 1 above)
Comment (nigelmegitt) [reply spec/entitiesedcopy.dtd]: OK, I've got caught out by this before. I wish there was a way to say WD (but by the way this is an ED)!

**Pull request merged**